FirebaseCombineSwift/Sources/Auth/GameCenterAuthProvider+Combine.swift

```diff
@@ -18,15 +18,15 @@
   import FirebaseAuth
 
   @available(swift 5.0)
+  @available(macOS 10.15, macCatalyst 13, iOS 13, tvOS 13, *)
```

Comment:
Good spot!

It looks like we're generally inconsistent between 13 and 13.0 across the repo.

Reply:
I just looked and Apple uses **13.0** in their docs. I'll update this and elsewhere in Combine APIs to follow that convention.